src/include/sof/audio/drc/drc_math.h

```diff
@@ -0,0 +1,73 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2020 Google LLC. All rights reserved.
+ *
+ * Author: Pin-chih Lin <johnylin@google.com>
+ */
+#ifndef __SOF_AUDIO_DRC_DRC_MATH_H__
+#define __SOF_AUDIO_DRC_DRC_MATH_H__
+
+#include <math.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#define DRC_PI_FLOAT 3.141592653589793f
+#define DRC_PI_OVER_TWO_FLOAT 1.57079632679489661923f
+#define DRC_TWO_OVER_PI_FLOAT 0.63661977236758134f
+#define DRC_NEG_TWO_DB 0.7943282347242815f /* -2dB = 10^(-2/20) */
+
+#ifndef max
+#define max(a, b)                                                              \
+	({                                                                     \
+		__typeof__(a) _a = (a);                                        \
+		__typeof__(b) _b = (b);                                        \
+		_a > _b ? _a : _b;                                             \
+	})
+#endif
+
+#ifndef min
+#define min(a, b)                                                              \
+	({                                                                     \
+		__typeof__(a) _a = (a);                                        \
+		__typeof__(b) _b = (b);                                        \
+		_a < _b ? _a : _b;                                             \
+	})
+#endif
+
+static inline float decibels_to_linear(float decibels)
```

Comment:
There's a version in sof/math/decibels.h. Though I'm not sure if fit your needs. To achieve fast speed it's not as accurate expf(). The other way for lin2db is missing from there. If you plan to do a fractional version contributing there is welcome!